client/logic/ui/round_label.gd

```diff
@@ -4,7 +4,8 @@ extends Label
 @export var player_name: String:
 	set(value):
 		player_name = value
-		text = "%s %s %s" % [tr("PLAYER"), str(value), tr("TURN")]
+		# text = "%s %s %s" % [tr("PLAYER"), str(value), tr("TURN")]
+		text = tr("PLAYER_TURN", "Round announcement, showing which player's turn it is.").format({player_name= value})
```

Comment:
I'd drop the context, making it general, and move the removed text to the Weblate comment. The ID could also be changed to `ANNOUNCEMENT_PLAYER_TURN` if you think the current one is too general.

Reply:
Fix!